test-utils/src/main/java/com/redhat/service/smartevents/test/resource/PostgresResource.java

```diff
@@ -9,7 +9,7 @@
 
 public class PostgresResource implements QuarkusTestResourceLifecycleManager {
 
-    private final PostgreSQLContainer postgres = new PostgreSQLContainer<>("postgres:10.5")
+    private final PostgreSQLContainer postgres = new PostgreSQLContainer<>("postgres:13.1")
```

Comment:
no need to do this in this PR but maybe it would be good to have this moved to a maven properties so it can be more visible and setup dynamically